### DISCUSSION TITLE
[Markdown] HTML tags <abbr> and <dfn> aren't rendering in GFM

### DISCUSSION THREAD

**Discussion by samkhan1** [community/community#46669] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

The following expected behaviour of html tags in GFM isn't working -

`<abbr title="Github Flavoured Markdown">GFM</abbr>` needs to render the tagged text with dots bellow it, which should show the specified title as a tip upon hovering over the tagged text. 

Similarly, the `<dfn>` tag isn't being rendered properly in GFM.   

Refernce:
- https://www.w3schools.com/tags/tag_abbr.asp
- https://www.w3schools.com/tags/tag_dfn.asp

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @samkhan1 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by ju1ius** [comment]
References to the real [HTML specification](https://html.spec.whatwg.org/multipage/) are always better than w3schools 😉 !

* [`<abbr>`](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-abbr-element)
* [`<dfn>`](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-dfn-element)

In addition to that, support for the `dfn[id]` attribute would be much welcome so that we can:

```markdown
Define <dfn id="the-term">a term</dfn>...
...and later on link to the definition of [this term](#the-term).
```

**Reply by beedell-roke** [reply]
@ju1ius and @samkhan1, [`talk.commonmark.org/t/3701`](https://talk.commonmark.org/t/dfn-the-define-element/3701/6?u=rokejulianlockhart) discusses adding this to CommonMark, in lieu of more comprehensive WHATWG HTML5 LS support in GHFM. Regardless, in lieu of that, I have filed [`github/markup/issues/2044`](https://github.com/github/markup/issues/2044#issue-4278225037). However, note [`github-linguist/linguist/issues/7637#issuecomment-3411091064`](https://github.com/github-linguist/linguist/issues/7637#issuecomment-3411091064)…

**Comment by shadowplumber** [comment]
I have this same problem with the exact same two HTML elements.
